twitter-user-feed.js

```diff
@@ -61,6 +69,10 @@ H5P.TwitterUserFeed = (function ($) {
     }(document, "script", "twitter-wjs"));
   };
 
+  // Inheritance
+  C.prototype = Object.create(H5P.EventDispatcher.prototype);
```

Comment:
This overwrites the prototype object, so that e.g. attach is no longer present. Fixed this one here:
https://github.com/h5p/h5p-twitter-user-feed/commit/5307e096e